### PR TITLE
Fix #455 Support only classpath: XSLT URIs

### LINKED ISSUE #455: Support only classpath: XSLT URIs
This is a consequence of https://github.com/apache/camel-quarkus/issues/439#issuecomment-557021163

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/BuildTimeUriResolver.java [hunk 1]
```diff
@@ -48,42 +37,26 @@ public ResolutionResult resolve(String templateUri) throws TransformerException
...
-        } else if ("classpath:".equals(effectiveScheme)) {
-            final String compacted = compact(templateUri, scheme);
-            transletName = toTransletName(compacted);
-            final URL url = getClass().getClassLoader().getResource(compacted);
-            try {
-                src = new StreamSource(url.openStream());
-            } catch (IOException e) {
-                throw new TransformerException("Could not read the class path resource " + templateUri, e);
-            }
+        if (!"classpath:".equals(effectiveScheme)) {
```
Comment (lburgazzoli) [inline extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/BuildTimeUriResolver.java]: afaik, no scheme is also a synonymous of class path

Comment (lburgazzoli) [reply extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/BuildTimeUriResolver.java]: sorry, missed the line above

Comment (ppalaga) [reply extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/BuildTimeUriResolver.java]: Yes, that's covered by the lines above https://github.com/apache/camel-quarkus/pull/456/files/ecd911af570372f846b242c597f73c9b6ce1c91f#diff-a4d51312ef4951bcb880ba4e9e863d0eR37-R38

**Comment by davsclaus** [comment]
ok to test

**Pull request merged**